src/VisualStudio/CSharp/Test/CSharpVisualStudioTest.csproj

```diff
@@ -216,10 +216,6 @@
     <Compile Include="ProjectSystemShim\CPS\SourceFileHandlingTests.cs" />
     <Compile Include="ProjectSystemShim\LegacyProject\SourceFileHandlingTests.cs" />
     <Compile Include="ProjectSystemShim\TempPECompilerServiceTests.cs" />
-    <Content Include="..\..\Setup\e_sqlite3.dll">
```

Comment:
You are probably still going to want this for tests (but with the x86/x64 directories).